src/index.js

```diff
@@ -22,16 +22,10 @@ if (
 }
 
 ReactDOM.render(
-  <Root.Provider>
-    <PublicUrl.Provider url="./aragon-ui/">
-      <BaseStyles />
-
-      <GlobalErrorHandler>
-        <ToastHub>
-          <App />
-        </ToastHub>
-      </GlobalErrorHandler>
-    </PublicUrl.Provider>
-  </Root.Provider>,
+  <Main>
+    <GlobalErrorHandler>
```

Comment:
Would it make sense to integrate the `GlobalErrorHandler` Component in `Main`?

Reply:
It would, but `GlobalErrorHandler` is using the `PublicUrl.Provider` and `BaseStyles`. But thinking again about it, I could include only these two components in `GlobalErrorHandler` itself, to only use the minimal amount of components needed there. 👍